### ISSUE TITLE
Solution for drag puzzle captcha

### ISSUE THREAD

**Issue by yudataguy** [StevanFreeborn/nytimes-sub-renewal#1]
Thank you for the template, I had the same idea of renewing NYT via library automatically, but wasn't familiar with playwright. Have you had the issue with drag puzzle captcha? Do you have any solution? Even though I manually dragged puzzle, it still think I'm a bot in the end. 

I want to build the same process for WSJ and other publications. But the bot detection is blocking my efforts.

**Comment by StevanFreeborn** [comment]
Yeah, there is some new trickery that is being done for bot detection that I myself have yet to come up with a good solution. Bypassing the captcha doesn't work because they are leveraging multiple bot detection strategies.

There is a lot of discussion around it [here](https://github.com/berstend/puppeteer-extra/issues/899).

And the exact detection approach is described in [this blog post](https://datadome.co/threat-research/how-new-headless-chrome-the-cdp-signal-are-impacting-bot-detection/).

I made some attempts at bypassing it by using some injected scripts that counter how they are modifying the `stack` getter, but nothing successful. I set it down a bit ago. Need to revisit.

This is a good website to test any bypass ideas you have or want to try: https://www.browserscan.net/bot-detection
